apps/aecore/lib/aecore/governance/governance_constants.ex

```diff
@@ -56,4 +75,31 @@ defmodule Aecore.Governance.GovernanceConstants do
 
   @spec claim_expire_by_relative_limit :: non_neg_integer()
   def claim_expire_by_relative_limit, do: @claim_expire_by_relative_limit
+
+  @spec genesis_prev_hash :: binary()
```

Comment:
I think removing the "genesis_" out of these functions will read better, maybe from the constants too

Reply:
For the constants I think it would be clearer if they contain "genesis" in their names, since they are part of the Governance module.

Reply:
I think it would be good if we move them out as well

https://github.com/aeternity/elixir-node/pull/532#discussion_r209966933

Reply:
@thepiwo So, maybe move them to the config ?

Reply:
no, to a separate constants file, or the governance module

Reply:
@thepiwo They are in the Governance module

Reply:
Sorry for the misscommunication, I was thinking of a separate governance file for genesis constants

Reply:
Will be done.